README.md

```diff
@@ -206,4 +206,6 @@ The pipeline adds several labels:
 * You also need to provide the `DOCKER_IMAGE` secret specifying your Docker image name, e.g., `quay.io/[username]/nfs-subdir-external-provisioner`.
 
 
-
+## NFS limitations
+* The provisioned storage limit is not enforced. An application can expand to use all the available storage regardless of the provisioned size.
```

Comment:
the storage size is also not guaranteed. can you add something about it?

Reply:
Good point. I tried adding a claim now.